.travis.yml

```diff
@@ -4,9 +4,14 @@ go: "1.10"
 services:
 - docker
 
+before_install:
+- go get golang.org/x/tools/cmd/goimports
```

Comment:
if we download and compile on travis can you please update the version to go1.11?
https://github.com/kubevirt/cluster-network-addons-operator/pull/131/files#diff-354f30a63fb0907d4ad57269548329e3R2

Reply:
Done